details_land.md

```diff
@@ -32,23 +32,23 @@ Finally, because the logit sharing approach reflects non-linear representations
 
 The other extreme is that of no substitution. This would be accomplished with near zero or zero logit exponents, whether in a single nest or multiple nests. This implies that either it is physically impossible or the transition costs are too high to allow switching from one land type to another. Currently, in GCAM, we use zero logit exponents in a very limited number of situations, where we do not want any substitution (e.g., we do not allow cropland to expand into desert or tundra).  However, in most situations, we employ positive logit exponents and allow economics to dictate the land allocation within a region. 
 
-Our approach is to use a nesting strategy that allows the logit exponents to reflect differences in substitutability across land categories. Figure 1 shows the nesting diagram of land with a subregion. At the top is all land, which is divided into two main types of nodes: agro-forestry land and the remaining categories of land that are not suitable for agriculture. This second category could be divided further if useful. The next node layer contains two further nodes: all agro-forestry, non-pasture land and all pasture land. The pasture land node contains two competing uses (land leaves in the code): managed pasture (that which feeds marketed livestock) and unmanaged pasture. The figure also indicates which file in the `gcamdata` system processes the inputs for each type of land (e.g., processing for purple leafs is found in [zchunk_L2242.land_input_4_irr_mgmt.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/zchunk_L2242.land_input_4_irr_mgmt.R) and [zchunk_batch_land_input_4_IRR_MGMT_xml.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/zchunk_batch_land_input_4_IRR_MGMT_xml.R), with the final inputs set in a file called `land_input_4_IRR_MGMT.xml`).
+Our approach employs a nesting strategy to capture the varying substitutability across different land categories. Figure 1 illustrates the nesting diagram of land within a subregion. The hierarchy begins with all land, which is then divided into two main nodes: agro-forestry land and non-agricultural land categories. The latter category can be further subdivided if necessary. The next layer includes two nodes: non-pasture land and pasture land. The pasture land node comprises two competing uses: managed pasture (grazed land for livestock) and unmanaged pasture. The agro-forestry (non-pasture) node consists of three competing nodes: shrub and grasslands, forest lands, and croplands. Shrublands and grasslands are treated separately as unmanaged land categories to control their substitutability. The forestland node competes with the total cropland node. Within forestland, managed and unmanaged forest leaves are considered, with the addition of woody biomass options in specific regions and scenarios. It is important to note that biomass crops are introduced into the model in 2025, as they are not included in the base data.
```

Comment:
`with the addition of woody biomass options in specific regions and scenarios`:  why is this scenario specific?

Reply:
removed, and a couple of sentences were added there.